src/dnnl/dnnl_graph_compiler.cpp

```diff
@@ -1,9 +1,21 @@
 #include "dnnl_graph_compiler.hpp"
+#include "dnnl_graph_compiler_version.h"
 #include <new>
 
 // dnnl_graph_compiler.h interface implementation.
 // TODO: Implement
 
+dnnl_status_t
+dnnl_graph_compiler_get_api_version(dnnl_graph_compiler_api_version *v) {
+  if (!v)
+    return dnnl_invalid_arguments;
+  v->major = GC_API_VERSION_MAJOR;
+  v->minor = GC_API_VERSION_MINOR;
+  v->patch = GC_API_VERSION_PATCH;
+  v->hash = GC_API_VERSION_HASH;
```

Comment:
What is VERSION_HASH? Is it the git hash? What's the purpose of this field?

Reply:
yes, this is a git commit hash, I thought it would be useful to see it in logs

Reply:
> What should the value be if it's built not from the git tree?

it's "N/A" in that case

> To be honest, I don't like the build time generated headers, it overcomplicates, for example, the source code integration.

Well, oneDNN already uses generated headers, so adding one more shouldn't be a problem right? :D